docs/pages/database-access/troubleshooting.mdx

```diff
@@ -137,3 +137,18 @@ Now Alice can connect to any database in the Teleport cluster using any database
 This example is intentionally simple; we could have configured Alice's permissions using more fine-grained control.
 For more detailed information about database access controls and how to restrict
 access see the [RBAC](../database-access/rbac.mdx) documentation.
+
```

Comment:
the troubleshooting.mdx is included in all kinds of databases. not sure if it will be confusing showing MySQL database error there or if there an easier way to hide/show this (scope?)

Reply:
The `database-access/troubleshooting.mdx` is used only in pages/database-access/getting-started.mdx:(!docs/pages/includes/database-access/aws-troubleshooting.mdx!) and 
```
     {
          "title": "Troubleshooting",
          "slug": "/database-access/troubleshooting/"
        }
```
```
rg 'troubleshooting.mdx' | grep -v "kubernetes-access" | grep -v "access-controls" | grep -v "desktop-access" | grep -v "deploy-a-cluster" | grep -v "machine-id"
pages/management/admin.mdx:- [Troubleshooting](./admin/troubleshooting.mdx): Collect metrics and diagnostic information from Teleport.
pages/database-access/introduction.mdx:documentation](./troubleshooting.mdx) for common problems and solutions.
pages/database-access/getting-started.mdx:(!docs/pages/includes/database-access/aws-troubleshooting.mdx!)
pages/database-access/guides/postgres-redshift.mdx:(!docs/pages/includes/database-access/aws-troubleshooting.mdx!)
pages/database-access/guides/rds.mdx:(!docs/pages/includes/database-access/aws-troubleshooting.mdx!)
pages/database-access/guides/redis-aws.mdx:(!docs/pages/includes/database-access/aws-troubleshooting.mdx!)
pages/database-access/guides/azure-redis.mdx:(!docs/pages/includes/database-access/azure-troubleshooting.mdx!)
pages/database-access/guides/azure-postgres-mysql.mdx:(!docs/pages/includes/database-access/azure-troubleshooting.mdx!)
pages/database-access/guides/rds-proxy.mdx:(!docs/pages/includes/database-access/aws-troubleshooting.mdx!)
pages/database-access/guides/redshift-serverless.mdx:(!docs/pages/includes/database-access/aws-troubleshooting.mdx!)

where other docs integration for most cases refers to `aws-troubleshooting.mdx!`